ios/ADYRedirectComponent.m

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+#import "ADYRedirectComponent.h"
+#import <adyen_react_native-Swift.h>
+
+@implementation ADYRedirectComponent
+
++ (BOOL)applicationOpenURL:(nonnull NSURL *)url {
```

Comment:
The m file is still the old one :D

Reply:
Fixed [872c374](https://github.com/Adyen/adyen-react-native/pull/121/commits/872c374f829dc66144a03fe645f63554a6ef51e3). Thanks!